specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json

```diff
@@ -0,0 +1,4002 @@
+        "x-ms-long-running-operation": true,
+        "tags": [
+          "WorkloadNetworks"
+        ],
+        "summary": "Create a segment by id in a private cloud workload network.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/privateCloudName"
+          },
+          {
+            "$ref": "#/parameters/segmentId"
+          },
+          {
+            "name": "workloadNetworkSegment",
+            "description": "NSX Segment",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/WorkloadNetworkSegment"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/WorkloadNetworkSegment"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "WorkloadNetworks_CreateSegments": {
+            "$ref": "./examples/WorkloadNetworks_CreateSegments.json"
+          }
+        }
+      },
+      "patch": {
+        "operationId": "WorkloadNetworks_UpdateSegments",
+        "x-ms-long-running-operation": true,
+        "tags": [
+          "WorkloadNetworks"
+        ],
+        "summary": "Create or update a segment by id in a private cloud workload network.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/privateCloudName"
+          },
+          {
+            "$ref": "#/parameters/segmentId"
+          },
+          {
+            "name": "workloadNetworkSegment",
+            "description": "NSX Segment",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/WorkloadNetworkSegment"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
async PATCH uses 202 + location header.  https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#updating-using-patch

Reply:
I updated this PATCH to use a 202 and now LintDiff is giving the error:
`R2005 - LongRunningResponseStatusCode
A 'PATCH' operation 'WorkloadNetworks_UpdateSegments' with x-ms-long-running-operation extension must have a valid terminal success status code 200 or 201.
  at specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json:1740
`
Should I change it back or is this a bug in LintDiff?